internal/provider/data_source_certificate_test.go

```diff
@@ -163,46 +163,33 @@ func TestAccDataSourceCertificate_TerraformIO(t *testing.T) {
 // It can potentially break over time, and we will need to keep the
```

Comment:
nit: Do we still need this note if we're no longer using a live certificate?

Reply:
Ha, nope, thank you will remove.